### PR TITLE
Customer Invoice API updates.

### PR DESCRIPTION
**Pull request by kaleposhobios**

### CODE REVIEW

**Comment by kaleposhobios** [comment]
@r38y here's the latest, let me know what you think

**Comment by nathanmadams** [comment]
@kaleposhobios @r38y I updated this branch to match changes discussed during implementation.

File: brokers/rest.md [hunk 1]
```diff
@@ -758,10 +758,10 @@ Response:
 
 ## Create + Update Customers
 
-PUT https://api.hubtran.com/tms/customers/:external_id
+POST https://api.hubtran.com/tms/customers/:external_id
```
[the remark this answers is not in the document]
Comment (nathanmadams) [reply brokers/rest.md:761]: Yeah I like PUTs for upserts too.

[the remark this answers is not in the document]
Comment (nathanmadams) [reply brokers/rest.md:761]: Will update.

[the remark this answers is not in the document]
Comment (nathanmadams) [reply brokers/rest.md]: I changed the customer_invoices endpoint to match the PUT in the customers endpoint. And I've added support for PUT on the customer_invoices endpoint in my hubtran-web branch: https://github.com/HubTran/hubtran-web/pull/6394 (This is in addition to POST so it shouldn't break any existing POSTers)

**Comment by jorycw** [comment]
@r38y @kaleposhobios 
Is this update to carriers documentation right?  I'm having the builder fail with invoicing in the params, and looking at the title of this PR leads me to believe this should be stuck under Customer Invoice API?  Not Carriers API.

**Comment by jorycw** [comment]
> @r38y @kaleposhobios
> Is this update to carriers documentation correct? I'm having the builder fail with invoicing in the params, and looking at the title of this PR leads me to believe this should be stuck under Customer Invoice API? Not Customers API.

Misread something while just going off of git blame.  Hold off for now, looking deeper.

**Pull request merged**